### PR TITLE
Document fix subcommand

### PR DESCRIPTION
**Pull request by mnarus**
Updated README to include `fix` subcommand
Updated Coding-Conventions.md to signify which rules are covered by `fix` subcommand

### CODE REVIEW

File: docs/Coding-Conventions.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ This document is applicable to all Python versions which are not end-of-life.
 ### [F.1.1] ✔️ **DO** Use `black` to format your code 💻
 
 > 💻 This rule is enforced by error code BLK100
+ 
+> ✨ This rule is covered by running `ni-python-stylgeuide fix`
```
Comment (mshafer-NI) [inline docs/Coding-Conventions.md:66]: Can we say "automatically fixed"?

**Comment by mshafer-NI** [comment]
Black now fixes D.1.12, right?

**Comment by mshafer-NI** [comment]
Might be the same for D.1.11

File: docs/Coding-Conventions.md (hunk 1 above)
Comment (irwand) [inline docs/Coding-Conventions.md:66]:
```suggestion
> ✨ This rule is covered by running `ni-python-styleguide fix`
```

**Review by irwand — APPROVED** [review]

File: docs/Coding-Conventions.md (hunk 1 above)
Comment (mnarus) [reply docs/Coding-Conventions.md]: Done.

**Pull request merged**